spec/classes/rabbitmq_spec.rb

```diff
@@ -288,6 +288,26 @@
         }
       end
 
+      describe 'does not contain pre-ranch settings with default config' do
```

Comment:
If the default is true then you would expect it to be present, correct?

Reply:
Well, I agree that the wording could possibly use some improvement, but no, the setting suppresses certain pre-ranch config settings with newer versions of RabbitMQ.

I guess we could invert the boolean and use `config_pre_ranch`, but that also seems kind of awkward.

Reply:
Oh you're right, I was looking at the tests and misread that they both test the same thing. Now I see one is ```without_content``` and the other ```with_content```